src/outfit.ts

```diff
@@ -340,6 +342,108 @@ function cheeses(embezzlerUp: boolean) {
       )
     : [];
 }
+
+function mafiaThumbRing(equipMode: BonusEquipMode) {
+  if (!have($item`mafia thumb ring`) || ["free", "dmt"].some((mode) => mode === equipMode)) {
+    return new Map<Item, number>([]);
+  }
+
+  return new Map<Item, number>([
+    [$item`mafia thumb ring`, (1 / 0.96 - 1) * get("valueOfAdventure")],
+  ]);
+}
+
+function luckyGoldRing(equipMode: BonusEquipMode) {
+  // Ignore for DMT, assuming mafia might get confused about the volcoino drop by the weird combats
+  if (!have($item`lucky gold ring`) || ["dmt"].some((mode) => mode === equipMode)) {
```

Comment:
```suggestion
  if (!have($item`lucky gold ring`) || equipMode === "dmt") {
```

I'm assuming this is a remnant from `isFree`, but it's still confusing to read even if it's paralleling something else